3_core_core_analysis/9_core_gene_vs_essential_gene.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>This is a cool analysis. Not for this review, but just curious, did you look at the identity of the 212 that are not essential? just wondering if for example all of the ribosomal genes are in the shared fraction or not, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/core-accessory-interactome/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Good question. It looks like most ribosome genes are shared but there are a few that are not.</p>